_data/communication.yml

```diff
@@ -13,6 +13,13 @@ websites:
       sms: Yes
       doc: https://help.37signals.com/sign-in/questions/460-how-do-i-set-up-and-use-phone-verification-for-logging-in
 
+    - name: Carbon Black Community Exchange
+      url: https://community.carbonblack.com/
```

Comment:
Sorry for the late response.  It's been a pretty crazy few weeks for me.
I certainly could; however, that is a completely different platform, and I don't know that there is anything on their main site that should require a login.  The community site however does have information that would cause some pretty significant security concerns if it were breached (which is why I submitted it).  Thoughts?